source/MaterialXGenShader/ShaderGraph.h

```diff
@@ -161,6 +161,10 @@ class ShaderGraph : public ShaderNode
 
     // Temporary storage for outputs that require color transformations
     std::unordered_map<ShaderOutput*, ColorSpaceTransform> _outputColorTransformMap;
+  private:
+    bool _bakeTexture = false;
```

Comment:
Does the shader graph need its own copy of this variable, or can we access the GenOptions copy when it's needed during shader generation?